README.md

```diff
@@ -38,3 +39,9 @@ Please cite this publication in any work using our library:
 
 * **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
 
+### Acknowledgments
+
+The **GrPPI** library has been partially supproted by:
+
+* Project ICT 644235 *"REPHRASE: REfactoring Parallel Heterogeneous Resource-aware Applications"* funded by the European Comision through H2020 program (2015-2018).
```

Comment:
Replace `comision` by `commission`.

Please, consider the use of `ispell` before committing any document to the repository.